.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+<!--- TITLE -->
+
+#### Module/Algorithm
+- [ ] Core algorithm/module implementation
+- [ ] [Query module](https://memgraph.com/docs/memgraph/reference-guide/query-modules) implementation
+- [ ] Unit tests
+- [ ] End-to-end tests
+- [ ] Code documentation
+- [ ] README short description
+- [ ] Documentation on [memgraph/docs](https://github.com/memgraph/docs)
+
+#### Bug/Enhacement
+
+<!--- Write the problem description and propose how to solve it -->
+
+- [ ] Code documentation
+- [ ] Working E2E tests
```

Comment:
Since E2E tests only apply to query modules, do we need to require them for potential contributions on, say, the API?

Reply:
That's true since the code is checked by Github Action on each PR push.

Reply:
Therefore, merging is blocked before actions pass.